bugbug/models/spamcomment.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+import xgboost
+from imblearn.over_sampling import BorderlineSMOTE
+from imblearn.pipeline import Pipeline as ImblearnPipeline
+from sklearn.compose import ColumnTransformer
+from sklearn.feature_extraction import DictVectorizer
+from sklearn.pipeline import Pipeline
+
+from bugbug import bugzilla, comment_features, db, feature_cleanup, repository, utils
+from bugbug.model import CommentModel
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+SAFE_DOMAINS = {"github.com", "mozilla.com", "mozilla.org"}
+
+
+class SpamCommentModel(CommentModel):
+    def __init__(self, lemmatization=True):
+        CommentModel.__init__(self, lemmatization)
+
+        self.calculate_importance = False
+
+        self.use_scale_pos_weight = True
+
+        feature_extractors = [
+            comment_features.NumberOfLinks(SAFE_DOMAINS),
+            comment_features.WordCount(),
+            comment_features.HourOfDay(),
+            comment_features.DayOfYear(),
+            comment_features.WeekOfYear(),
```

Comment:
Since `DayOfYear` and `WeekOfYear` are highly correlated, did you check if using both of them is improving the results?

Reply:
I think one of them would work for us. Let me experiment and get back to you with a better response.